### PR TITLE
Added Flood Fill Problem

### LINKED ISSUE #6800: Flood Fill Problem in C++
## Description
Documentation of Flood Fill Problem in C++

### Domain

C/CPP

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by aman12207**
<hr>

## Description 📜

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
Fixes #6800 
<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] C/CPP
<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by Suvraneel** [comment]
@vasu-1

File: C_CPP/Algorithmic Approaches/Backtracking/Flood_Fill_Problem.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+Only four moves are allowed: 't' (1-step up), 'l' (1-step left), 'd' (1-step down) 'r' (1-step right) in the given order only.
+You are required to print all paths that can be used to move from top-left to bottom-right.
+
+## Approach
+* Here, we pass an empty boolean 2d array in the function as a parameter called "visited[ ][ ]" which has the dimensions same as are our input array.
+* Before visiting a cell in the maze array , we mark the corresponding cell in the visited array " true" so that in the duration of the same path we don't visit the already visited cell again.
+
+## CODE
+
+```c
```
Comment (apeksha235) [inline C_CPP/Algorithmic Approaches/Backtracking/Flood_Fill_Problem.md:13]: cpp* not c

File: C_CPP/Algorithmic Approaches/Backtracking/Flood_Fill_Problem.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+#include<iostream>
+#include<vector>
+
+using namespace std;
+const int col = 11;
+
+void floodfill(vector<vector<int>> maze,int rows,int columns,string ans,bool visited[][col]) {
```
Comment (apeksha235) [inline C_CPP/Algorithmic Approaches/Backtracking/Flood_Fill_Problem.md:20]: please add relevant comments to your code

**Review by apeksha235 — CHANGES_REQUESTED** [review]
@aman12207

**Comment by aman12207** [comment]
I made the changes as directed..

**Comment by Suvraneel** [comment]
@aman12207  Do these & tag me

**Comment by aman12207** [comment]
@Suvraneel Please tell me what is need to do..

**Review by Suvraneel — CHANGES_REQUESTED** [review]

File: C_CPP/Algorithmic Approaches/Backtracking/Flood_Fill_Problem.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
+# Problem Statement
+You are given a 2d array of dimensions n*m.
+This array is filled with either 0 or 1. You can only move to cells which have 0 value in them. You can't move out of the boundaries or in the cells which have value 1 in them (1 means obstacle)
```
[the remark this answers is not in the document]
Comment (aman12207) [reply C_CPP/Algorithmic Approaches/Backtracking/Flood_Fill_Problem.md:3]: Ohh.. Thanks a lot i don't know that before 
previously I am using br tags to do  the same

**Comment by Suvraneel** [comment]
@aman12207  LGTM

**Comment by aman12207** [comment]
> @aman12207 LGTM

Thanks..

**Pull request merged**